docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -236,20 +236,30 @@ ALTER TABLE table_identifier DROP [ IF EXISTS ] partition_spec [PURGE]
      
 ### SET AND UNSET
 
-#### SET TABLE PROPERTIES
+#### SET PROPERTIES
 
 `ALTER TABLE SET` command is used for setting the table properties. If a particular property was already set, 
 this overrides the old value with the new one.
 
-`ALTER TABLE UNSET` is used to drop the table property. 
-
 ##### Syntax
 
 ```sql
--- Set Table Properties 
+-- Set Properties
 ALTER TABLE table_identifier SET TBLPROPERTIES ( key1 = val1, key2 = val2, ... )
+```
+
+#### UNSET PROPERTIES
+
+`ALTER TABLE UNSET` command is used to drop the table property.
 
--- Unset Table Properties
+**Note:** If the specified property key does not exist, when you use the v1 command and do not specify `IF EXISTS`,
+it will throw the error-condition `UNSET_NONEXISTENT_PROPERTIES` and finally `failed`,
```

Comment:
shall we change the v1 behavior as well? It's not a breaking change to change from failure to no failure.

Reply:
and we should avoid mentioning v1/v2 in the doc. It's an internal concept and should not be exposed to end users

Reply:
> shall we change the v1 behavior as well? It's not a breaking change to change from failure to no failure.



> shall we change the v1 behavior as well? It's not a breaking change to change from failure to no failure.

Sure, let me update it.

Reply:
> and we should avoid mentioning v1/v2 in the doc. It's an internal concept and should not be exposed to end users

Okay.

Reply:
> shall we change the v1 behavior as well? It's not a breaking change to change from failure to no failure.

Done.